packages/commonwealth/client/scripts/views/components/component_kit/CWRelatedCommunityCard/CWRelatedCommunityCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
shouldn't we put this in new_designs folder?

Reply:
Yes! Moving it over!